.github/workflows/publish_models_dev.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish models to dev
+on:
+  push:
+    branches:
+      - dev
```

Comment:
I don't think there is a dev branch on this repo

Reply:
Thanks for the catch. Switched dev workflow back to main branch.